docarray/array/mixins/io/csv.py

```diff
@@ -86,6 +86,8 @@ def save_csv(
                 if flatten_tags:
                     t = pd.pop('tags')
                     pd.update({f'tag__{k}': v for k, v in t.items()})
+
+                pd = {k: str(v).replace('\n', ' ') for k, v in pd.items()}
```

Comment:
lets rename this to something else than `pd` since this is usually use for pandas

Reply:
What I don't understand is why the parser don't just do a return to the line by itself ? Is it because it does not handle it ?